src/WinGet.RestSource.Utils/WinGet.RestSource.Utils.csproj

```diff
@@ -44,7 +44,6 @@
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.22.0" />
     <PackageReference Include="Microsoft.WindowsPackageManager.Utils" Version="0.3.4" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.12" />
-    <PackageReference Include="Microsoft.WindowsPackageManager.Utils" Version="*" />

```

Comment:
"*" is actually preferred over specific version for Utils package so we can get the latest winget manifest object model. We could remove line 45 instead.

Reply:
FIxed